posthog/api/routing.py

```diff
@@ -236,11 +239,11 @@ def _is_project_view(self):
 
     @cached_property
     def team_id(self) -> int:
-        if self._is_project_view:
+        if self._is_project_view and not self._skip_team_id_override_kludge:
             team_id = self.project_id  # KLUDGE: This is just for the period of transition to project environments
         elif team_from_token := self._get_team_from_request():
             team_id = team_from_token.id
-        elif self.param_derived_from_user_current_team == "team_id":
+        elif self.param_derived_from_user_current_team == "team_id" or self._skip_team_id_override_kludge:
```

Comment:
This appears to be a bug: because of this being an `or` viewsets with `_skip_team_id_override_kludge = True` now ignore the `team_id` from the URL and instead always use `user.team`. That's not an intention signaled by `_skip_team_id_override_kludge`